src/request-data/user-preferences/normalizer.js

```diff
@@ -0,0 +1,36 @@
+const VUE_PROPERTY_PREFIX = '__v_'; // Empirically established. I couldn't find documentation on it.
+
+
+class PreferenceNotRegisteredError extends Error {
+  constructor(prop, whatclass, ...params) {
```

Comment:
> It's basically the way to prevent unnecessarily narrowing of the behaviour, keeping the way open for implementors to access present and future functionality of the Error parent class.

I personally think the code would be clearer if it only did what we needed it to. Currently, we don't need `PreferenceNotRegisteredError` to support a `cause` option or to support future `Error` functionality, so we could simplify it by removing `...params`. If we need it to do more in the future, it'd be easy to add to it then. What do you think?

I'm also wondering about the next line:

```js
super(...params);
```

`params` won't ever include a message, right? If we're going to call `super()` with arguments, I think we should pass it the message as the first argument.